releasenotes/notes/operator_revision.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - 23479
+releaseNotes: |
+  *Added* --revision flag to `istioctl operator init` and `istioctl operator remove` commands to support multiple control plane upgrade.
```

Comment:
I think what we mean is  "In place single Control Plane upgrade using Canary". Istio doesnt support out-of-box multiple control plane within a single cluster. It might be helpful to clarify this please.